src/schema/cruise-result.schema.js

```diff
@@ -167,8 +167,27 @@ module.exports = {
       additionalProperties: false,
       properties: {
         name: { type: "string" },
-        dependents: { type: "array", items: { type: "string" } },
-        dependencies: { type: "array", items: { type: "string" } },
+        dependents: {
```

Comment:
This (and all folder-related updates) should be in a separate PR as they're not contributing to the PR's direct goal